sigal/plugins/zip_gallery.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 - Remi Ferrand
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+""" This plugin controls the generation of a ZIP archive for a gallery
+
+To ignore a ZIP gallery generation for a particular album, put a ``.nozip_gallery`` file next to it in its parent
+folder. Only the existence of this ``.nozip_gallery`` file is tested.
```

Comment:
Could you mention here the `zip_gallery` setting, as this will be shown in the docs. Basically what's in the former `zip` docstring.

Reply:
I've adapted the _docstring_. Should be okay now.